package.json

```diff
@@ -88,7 +88,7 @@
     "screwdriver-build-bookend": "^2.0.1",
     "screwdriver-command-validator": "^1.0.2",
     "screwdriver-config-parser": "^3.14.1",
-    "screwdriver-data-schema": "^18.13.5",
+    "screwdriver-data-schema": "file:../data-schema",
```

Comment:
Will update this once the data-schema PR is reviewed and merged: https://github.com/screwdriver-cd/data-schema/pull/229